Library/Homebrew/cask/config.rb

```diff
@@ -15,6 +15,7 @@ class Config
 
     DEFAULT_DIRS = {
       appdir:               "/Applications",
+      keyboard_layoutdir:   "/Library/Keyboard Layouts",
```

Comment:
```suggestion
      keyboard_layoutdir:   "~/Library/Keyboard Layouts",
```

Reply:
This is intentional. You will need `sudo` to delete the cache anyways, so might as well install in the system-wide directory. Arguably, all of these should use the system-wide directory by default, but it would be pretty hard to migrate these now.